vach-cli/Cargo.toml

```diff
@@ -39,4 +36,6 @@ lazy_static = "1.4.0"
 tabled = "0.3.0"
 log = "0.4.14"
 pretty_env_logger = "0.4.0"
-vach = "0.2.2"
+vach = {	path = "../vach"	}
```

Comment:
we need to use a version availible on crates.io for it to be published

Reply:
I had to modify vach and add some utility functions to vach, the CLI depends on those features. Once I publish the latest version of vach. then I will switch to a pathless dependency